Neos.Flow/Classes/Security/Policy/PolicyService.php

```diff
@@ -115,9 +115,11 @@ protected function initialize(): void
                     $role = $everybodyRole;
                 } else {
                     $role = new Role($roleIdentifier);
-                    if (isset($roleConfiguration['abstract'])) {
-                        $role->setAbstract((boolean)$roleConfiguration['abstract']);
+                    $role->setAbstract((bool)($roleConfiguration['abstract'] ?? false));
+                    if (isset($roleConfiguration['label'])) {
+                        $role->setLabel((string)$roleConfiguration['label']);
                     }
```

Comment:
```suggestion
                    $role->setLabel((string)($roleConfiguration['label'] ?? $roleIdentifier));
```
Maybe to have those lines in line?

Reply:
That would mean that the default set in https://github.com/neos/flow-development-collection/pull/2166/files#diff-a6e6a5cf33830d6eb63ae27a24b440dcR92 would be overwritten in any case. I would prefer the split part there.